Rollbar/RollbarConfig.cs

```diff
@@ -0,0 +1,31 @@
+namespace RollbarDotNet
+{
+    public class RollbarConfig
+    {
+        public RollbarConfig() : this("")
+        {
+        }
+
+        public RollbarConfig(string accessToken)
+        {
+            AccessToken = accessToken;
+            Environment = "production";
+            Enabled = true;
+            LogLevel = ErrorLevel.Debug;
+            ScrubField = new[] { "passwd", "password", "secret", "confirm_password", "password_confirmation" };
```

Comment:
Here's a list of the default scrub fields for the python notifier - https://github.com/rollbar/pyrollbar/blob/master/rollbar/__init__.py#L211

and for the gem - https://github.com/rollbar/rollbar-gem/blob/master/lib/rollbar/configuration.rb#L98

You may want to factor them out of this class so they are easier to find/update.